testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -6188,6 +6188,29 @@ void checkRegexpExtract(SqlOperatorFixture f0, FunctionAlias functionAlias) {
     f.checkNull("log(10, cast(null as real))");
   }
 
+  @Test void testLog2Func() {
+    final SqlOperatorFixture f0 = Fixtures.forOperators(true);
+    f0.setFor(SqlLibraryOperators.LOG2, VmName.EXPAND);
+    f0.checkFails("^log2(4)^",
+        "No match found for function signature LOG2\\(<NUMERIC>\\)", false);
+    final SqlOperatorFixture f = f0.withLibrary(SqlLibrary.MYSQL);
```

Comment:
Even if the behavior does not vary across libraries, it is easy to test against all libraries using the `consumer` pattern. See any test that uses `forEachLibrary()` for examples.